### PR TITLE
rh7: Blacklist mlx ib drivers to prevent it from loading.

### PR DESCRIPTION
**Pull request by johnsongeorge-w**

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@santoshx @prashantmhetar Review this changes.

File: hv-rhel7.x/hv/rhel7-hv-driver-install [hunk 1]
```diff
@@ -17,6 +17,11 @@ if [ $rhel_release_code -ge 1795 ] && [ $rhel_release_code -lt 1798 ]; then
 	mkdir -p /lib/modules/$KERNEL_VERSION/extra/lis-mlx
 	find . -name *.ko -exec cp {} /lib/modules/$KERNEL_VERSION/extra/lis-mlx \;
 	cd ..
+
+	if [ -f ./hyperv_bl_drv.conf ];then
```
Comment (santoshx) [inline hv-rhel7.x/hv/rhel7-hv-driver-install]: This check for file existence is not needed.

File: hv-rhel7.x/hv/rhel7-hv-driver-uninstall [hunk 2]
```diff
@@ -14,6 +14,9 @@ rm -rf  /lib/modules/$KERNEL_VERSION/extra/uio_hv_generic.ko
 
 rm -rf  /lib/modules/$KERNEL_VERSION/extra/microsoft-hyper-v/hv_*.ko
 rm -rf /etc/modprobe.d/hyperv_pvdrivers.conf
+if [ -f /etc/modprobe.d/hyperv_bl_drv.conf ];then
```
Comment (santoshx) [inline hv-rhel7.x/hv/rhel7-hv-driver-uninstall]: This check for file existence is not needed.

File: hv-rhel7.x/hv/hyperv_bl_drv.conf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Do not load the 'mlx ib' drivers
```
Comment (santoshx) [inline hv-rhel7.x/hv/hyperv_bl_drv.conf]: I don't see a need for creating a separate file in repository.
Just using two lines echo from install script will suffice.

Comment (johnsongeorge-w) [reply hv-rhel7.x/hv/hyperv_bl_drv.conf]: Its better to have static file for better visibility.

**Comment by johnsongeorge-w** [comment]
@santoshx This is required for consistency and will enable local verification.
Removed file existence check as pointed and updated the comment.

**Pull request merged**